oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/datastore/DataStoreTextWriter.java

```diff
@@ -131,7 +130,7 @@ public void write(@NotNull String blobId,@NotNull String text) throws IOExceptio
         File textFile = getFile(stripLength(blobId));
         ensureParentExists(textFile);
         //TODO should we compress
-        Files.write(text, textFile, StandardCharsets.UTF_8);
+        org.apache.jackrabbit.guava.common.io.Files.write(text, textFile, StandardCharsets.UTF_8);
```

Comment:
shouldn't this be removed ?

Reply:
That's no functional change. We need a FQN here because JDK Files replaces Guava Files just for `Files.toString()`. Other methods will be replaced later on.